### PR TITLE
[MOB-4737] - Update README for getInAppMessages

### PR DESCRIPTION
**Pull request by ts-nguyen**
## JIRA Ticket(s) if any

https://iterable.atlassian.net/browse/MOB-4737

## Description

Adding documentation

## Test Steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,17 +102,23 @@ Close Button Options:
 
 Example:
 
+When using the method `getInAppMessages` with the `showInAppMessagesAutomatically` set to false or not set, the return value will be just the API response from the Iterable backend with the message data as the body. Inside there will be a list of inAppMessages and within each one the field `content.html` will be modified to be an instance of an `iframe` with the inAppMessage embedded inside. Iterable provides this `iframe` with the `sandbox` attribute set to provide an isolated render within a web application and to prevent any malicious JS execution.
```
Comment (bradumbaugh) [inline README.md:105]:
```suggestion
Calling `getInAppMessages` with `showInAppMessagesAutomatically` set to `false` (or not set) returns a JSON API response from Iterable. This response includes an `inAppMessages` field, and each item in the list has a `content.html` field that's an `iframe` with an embedded in-app message. The `iframe`'s `sandbox` attribute is set to `true`, isolating its render and preventing any malicious JavaScript execution.
```

File: README.md [hunk 2]
```diff
@@ -102,17 +102,23 @@ Close Button Options:
...
+      /* This will be an iframe element that can be attached to the DOM */
+      const messageIframe = resp.data.inAppMessages[0].content.html;
+      document.body.appendChild(messageIframe);
+
+      /* Additional styling logic can be done here to customly render the message */
+  })
   .catch()
 ```
 
+At this point the iframe with the message will be rendered on the page but not visible. The CSS will need to be modified to have the styling desired to render properly on the webpage as well as appropriate click handlers to close the message and send tracking events.
```
Comment (bradumbaugh) [inline README.md:120]:
```suggestion
This code places an in-app on the page, but it won't be visible.  To render it, you'll need to modify the page's CSS, setting up whatever styles you'd like. You'll also need to set up click handlers to handle closing the message and tracking events (in-app click, etc.).
```

File: README.md [hunk 3]
```diff
@@ -102,17 +102,23 @@ Close Button Options:
...
+
 or if you want to show messages automatically
```
Comment (bradumbaugh) [inline README.md:122]: @ts-nguyen Not sure what this line is?

**Review by bradumbaugh — CHANGES_REQUESTED** [review]
@ts-nguyen Left some suggestions and a question, let me know what you think!

File: README.md [hunk 4]
```diff
@@ -117,7 +117,7 @@ getInAppMessages({ count: 20, packageName: 'mySite1' })
   .catch()
 ```
 
-At this point the iframe with the message will be rendered on the page but not visible. The CSS will need to be modified to have the styling desired to render properly on the webpage as well as appropriate click handlers to close the message and send tracking events.
+This code places an in-app on the page, but it won't be visible.  To render it, you'll need to modify the page's CSS, setting up whatever styles you'd like. You'll also need to set up click handlers to handle closing the message and tracking events (in-app click, etc.).
 
 or if you want to show messages automatically
```
Comment (bradumbaugh) [inline README.md:122]: Ah, OK - might want to change this to:

```suggestion
Or, to show messages automatically:
```

File: README.md (hunk 3 above)
Comment (bradumbaugh) [reply README.md:122]: OK, left a suggestion below!

File: README.md (hunk 4 above)
Comment (bradumbaugh) [reply README.md:122]: Thanks @ts-nguyen, looks good to me! Appreciate you looping me in.

**Review by pauljung14 — APPROVED** [review]

**Pull request merged**